driver-core/src/main/com/mongodb/internal/operation/AggregateOperation.java

```diff
@@ -208,23 +208,37 @@ public Collation getCollation() {
     /**
      * Returns the comment to send with the aggregate. The default is not to include a comment with the aggregation.
      *
+     * <p>The comment can be any valid BSON type for server versions 4.4 and above.
```

Comment:
This seems like it was written for public consumption, but this is an internal class.

Can you clarify your intent?

Reply:
Happy to remove if required.

Reply:
I had it to document the behavior of strings vs other types. Will remove.